### PR TITLE
Periodic Table

### PR DESCRIPTION
**Pull request by yarm369**

### CODE REVIEW

File: src/periodic_table.py [hunk 1]
```diff
@@ -0,0 +1,359 @@
+METAL = 'Metal'
+NON_METAL = 'Non Metal'
+METALLOID = 'Metalloid'
+
+PERIODIC_TABLE = {
+    '1': {'atomic_num': 1, 'symbol': 'H', 'name': 'Hydrogen', 'group': 1, 'period': 1, 'atomic_weight': 1.008,
+          'density': 8.988e-05, 'melting_point': 14.01, 'boiling_point': 20.28, 'heat_capacity': 14.304,
+          'electronegativity': 2.2, 'precise': True, 'type': NON_METAL},
+    '2': {'atomic_num': 2, 'symbol': 'He', 'name': 'Helium', 'group': 18, 'period': 1, 'atomic_weight': 4.002602,
+          'density': 0.0001785, 'melting_point': None, 'boiling_point': 4.22, 'heat_capacity': 5.193,
```
Comment (isaacrivriv) [inline src/periodic_table.py:34]: Need to make note about electronegativity set to None to use in Element and Compound classes when comparing Elements

File: src/periodic_table.py [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+            'electronegativity': None, 'type': METAL},
+    '107': {'atomic_num': 107, 'symbol': 'Bh', 'name': 'Bohrium', 'group': 7, 'period': 7, 'precise': False,
+            'atomic_weight': 270, 'density': 37.1, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '108': {'atomic_num': 108, 'symbol': 'Hs', 'name': 'Hassium', 'group': 8, 'period': 7, 'precise': False,
+            'atomic_weight': 270, 'density': 40.7, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': METAL},
+    '109': {'atomic_num': 109, 'symbol': 'Mt', 'name': 'Meitnerium', 'group': 9, 'period': 7, 'precise': False,
+            'atomic_weight': 278, 'density': 37.4, 'melting_point': None, 'boiling_point': None, 'heat_capacity': None,
+            'electronegativity': None, 'type': None},
```
Comment (isaacrivriv) [inline src/periodic_table.py:356]: For type: None these are still under research right? They are all approximations right?

File: src/periodic_table.py [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+    '7': {'atomic_num': 7, 'symbol': 'N', 'name': 'Nitrogen', 'group': 15, 'period': 2, 'atomic_weight': 14.007,
+          'density': 0.0012506, 'melting_point': 63.15, 'boiling_point': 77.36, 'heat_capacity': 1.04,
+          'electronegativity': 3.04, 'precise': True, 'type': NON_METAL},
+    '8': {'atomic_num': 8, 'symbol': 'O', 'name': 'Oxygen', 'group': 16, 'period': 2, 'atomic_weight': 15.999,
+          'density': 0.001429, 'melting_point': 54.36, 'boiling_point': 90.2, 'heat_capacity': 0.918,
+          'electronegativity': 3.44, 'precise': True, 'type': NON_METAL},
+    '9': {'atomic_num': 9, 'symbol': 'F', 'name': 'Fluorine', 'group': 17, 'period': 2, 'atomic_weight': 18.998403163,
+          'density': 0.001696, 'melting_point': 53.53, 'boiling_point': 85.03, 'heat_capacity': 0.824,
+          'electronegativity': 3.98, 'precise': True, 'type': NON_METAL},
+    '10': {'atomic_num': 10, 'symbol': 'Ne', 'name': 'Neon', 'group': 18, 'period': 2, 'atomic_weight': 20.1797,
```
Comment (isaacrivriv) [inline src/periodic_table.py:57]: What corresponds to period and what corresponds to group in our case?

File: src/periodic_table.py [hunk 4]
```diff
@@ -0,0 +1,359 @@
...
+    '2': {'atomic_num': 2, 'symbol': 'He', 'name': 'Helium', 'group': 18, 'period': 1, 'atomic_weight': 4.002602,
+          'density': 0.0001785, 'melting_point': None, 'boiling_point': 4.22, 'heat_capacity': 5.193,
+          'electronegativity': None, 'precise': True, 'type': NON_METAL},
+    '3': {'atomic_num': 3, 'symbol': 'Li', 'name': 'Lithium', 'group': 1, 'period': 2, 'atomic_weight': 6.94,
+          'density': 0.534, 'melting_point': 453.69, 'boiling_point': 1560, 'heat_capacity': 3.582,
+          'electronegativity': 0.98, 'precise': True, 'type': METAL},
+    '4': {'atomic_num': 4, 'symbol': 'Be', 'name': 'Beryllium', 'group': 2, 'period': 2, 'atomic_weight': 9.0121831,
+          'density': 1.85, 'melting_point': 1560, 'boiling_point': 2742, 'heat_capacity': 1.825,
+          'electronegativity': 1.57, 'precise': True, 'type': METAL},
+    '5': {'atomic_num': 5, 'symbol': 'B', 'name': 'Boron', 'group': 13, 'period': 2, 'atomic_weight': 10.81,
```
Comment (isaacrivriv) [inline src/periodic_table.py:42]: Need to make note about what scale the boiling/melting point is in (F, K, C)

File: src/periodic_table.py [hunk 5]
```diff
@@ -0,0 +1,359 @@
...
+    '81': {'atomic_num': 81, 'symbol': 'Tl', 'name': 'Thallium', 'group': 13, 'period': 6, 'atomic_weight': 204.38,
+           'density': 11.85, 'melting_point': 577, 'boiling_point': 1746, 'heat_capacity': 0.129,
+           'electronegativity': 1.62, 'precise': True, 'type': METAL},
+    '82': {'atomic_num': 82, 'symbol': 'Pb', 'name': 'Lead', 'group': 14, 'period': 6, 'atomic_weight': 207.2,
+           'density': 11.342, 'melting_point': 600.61, 'boiling_point': 2022, 'heat_capacity': 0.129,
+           'electronegativity': 1.87, 'precise': True, 'type': METAL},
+    '83': {'atomic_num': 83, 'symbol': 'Bi', 'name': 'Bismuth', 'group': 15, 'period': 6, 'atomic_weight': 208.9804,
+           'density': 9.807, 'melting_point': 544.7, 'boiling_point': 1837, 'heat_capacity': 0.122,
+           'electronegativity': 2.02, 'precise': True, 'type': METAL},
+    '84': {'atomic_num': 84, 'symbol': 'Po', 'name': 'Polonium', 'group': 16, 'period': 6, 'precise': False,
```
Comment (isaacrivriv) [inline src/periodic_table.py:279]: Precise is the column we talked about when working with elements still under research which have some approximate values right?

File: src/periodic_table.py [hunk 6]
```diff
@@ -0,0 +1,359 @@
...
+           'atomic_weight': 209, 'density': 9.32, 'melting_point': 527, 'boiling_point': 1235, 'heat_capacity': None,
+           'electronegativity': 2.0, 'type': METAL},
+    '85': {'atomic_num': 85, 'symbol': 'At', 'name': 'Astatine', 'group': 17, 'period': 6, 'precise': False,
```
Comment (isaacrivriv) [inline src/periodic_table.py:282]: Since these are approximate, do they have an approximate atomic weight?

**Review by isaacrivriv — COMMENTED** [review]
Overall looks great. Just got a couple of questions to understand all the data corresponding to the elements

File: src/periodic_table.py (hunk 3 above)
Comment (yarm369) [reply src/periodic_table.py:57]: Period = row
Group = column

File: src/periodic_table.py (hunk 6 above)
Comment (isaacrivriv) [reply src/periodic_table.py:282]: Error on my part they all have

**Comment by isaacrivriv** [comment]
LGTM merging changes

**Pull request merged**